hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTable.java

```diff
@@ -632,16 +632,11 @@ private Boolean validateTable(final Component compsource, final TableTransferabl
         return true;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
This method is small but I would still move the `@SuppressWarnings("unchecked")` to line`638` where the actual unchecked operation occurs